### ISSUE TITLE
Error in custom_filename 7.0

### ISSUE THREAD

**Issue by ovnicraft** [OCA/reporting-engine#30]
i set ${o.name}.pdf to ewn file in report then try to print, i get this: 

``` python
2015-11-07 20:39:27,962 14890 ERROR sgp openerp.addons.web.http.HttpRequest.dispatch: An error occurred while handling a json request
Traceback (most recent call last):
  File "/home/odoo/odoo/addons/web/http.py", line 285, in dispatch
    r = method(self, **self.params)
  File "/home/odoo/reporting-engine/report_custom_filename/controllers/reports.py", line 46, in index
    generated_filename = email_template.mako_template_env\
  File "/home/odoo/.virtualenvs/esigpen/local/lib/python2.7/site-packages/jinja2/environment.py", line 969, in render
    return self.environment.handle_exception(exc_info, True)
  File "/home/odoo/.virtualenvs/esigpen/local/lib/python2.7/site-packages/jinja2/environment.py", line 742, in handle_exception
    reraise(exc_type, exc_value, tb)
  File "<template>", line 1, in top-level template code
  File "/home/odoo/.virtualenvs/esigpen/local/lib/python2.7/site-packages/jinja2/sandbox.py", line 330, in getattr
    value = getattr(obj, attribute)
  File "/home/odoo/odoo/openerp/osv/orm.py", line 503, in __getattr__
    return self[name]
  File "/home/odoo/odoo/openerp/osv/orm.py", line 406, in __getitem__
    field_values = self._table.read(self._cr, self._uid, ids, field_names, context=self._context, load="_classic_write")
  File "/home/odoo/odoo/openerp/osv/orm.py", line 3707, in read
    result = self._read_flat(cr, user, select, fields, context, load)
  File "/home/odoo/odoo/openerp/osv/orm.py", line 3761, in _read_flat
    cr.execute(query, [tuple(sub_ids)] + rule_params)
  File "/home/odoo/odoo/openerp/sql_db.py", line 160, in wrapper
    raise psycopg2.OperationalError(msg)
OperationalError: Unable to use a closed cursor.
```

This happens when render try to access to attribute object, debugging 

``` python
(Pdb) objects
[browse_record(prison.person, 69795)]
(Pdb) objects[0]
browse_record(prison.person, 69795)
(Pdb) objects[0].prontuario
*** OperationalError: Unable to use a closed cursor.
(Pdb) objects[0].name
*** OperationalError: Unable to use a closed cursor.
(Pdb) c
```

**Comment by StefanRijnhart** [comment]
Same here. Any clues @ovnicraft?

**Comment by ovnicraft** [comment]
@StefanRijnhart no reason found yet, only happens with 7.0 :(

**Comment by StefanRijnhart** [comment]
@ovnicraft could you test https://github.com/OCA/reporting-engine/pull/43

**Comment by pedrobaeza** [comment]
This was fixed in #43, so I close.
